README.md

```diff
@@ -15,4 +15,19 @@ cd ..
 ls
 ls -a
 
+git add .
+$ git commit -m <commit msg>
+
+$ git fetch
+$ git rebase origin/develop
+
+$ git push --force-with-lease origin <your branch>
```

Comment:
I don't get the word "--force-with-lease". what is this shatabdi?

Reply:
This is an interesting stuff.